packages/create-swarmion-app/package.json

```diff
@@ -35,7 +35,7 @@
     "test": "pnpm test-linter && pnpm test-type && pnpm test-unit && pnpm test-circular",
     "test-circular": "pnpm depcruise --validate .dependency-cruiser.js .",
     "test-linter": "pnpm linter-base-config .",
-    "test-type": "tsc --noEmit --emitDeclarationOnly false",
+    "test-type": "tsc",
```

Comment:
why don't we do this modification in every package ?

Reply:
To make `tsc --build` work in both packages and apps, we need to differentiate.
In apps, we will have `"test-type": "tsc",` and `"test-type": "tsc --noEmit --emitDeclarationOnly false",` in packages.